spec/features/actors_feature_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe "Actors", type: :feature do
+
+  before do
```

Comment:
In general, I like to avoid `before` and `let`, to avoid [Mystery Guests]().

Ref:

 - https://robots.thoughtbot.com/mystery-guest
 - https://robots.thoughtbot.com/lets-not